org.eclipse.sisu.plexus/build.target

```diff
@@ -17,19 +17,19 @@
 		<location type="Maven" includeSource="true">
 			<groupId>org.slf4j</groupId>
 			<artifactId>slf4j-api</artifactId>
-			<version>1.6.4</version>
+			<version>1.7.36</version>
```

Comment:
Let us remove the dependency changes as they are potentially backwards incompatible (affects import package bundle ranges).

Reply:
Undone